modules/permission-management/src/Volo.Abp.PermissionManagement.Blazor/Components/PermissionManagementModal.razor.cs

```diff
@@ -317,4 +328,26 @@ protected virtual bool IsPermissionGroupDisabled(PermissionGroupDto group)
 
         return permissions.All(x => x.IsGranted) && grantedProviders.Any(p => p.ProviderName != _providerName);
     }
+    
+    protected virtual async Task OnPermissionGroupSearchTextChangedAsync(string value)
+    {
+        if (value == _permissionGroupSearchText)
+        {
+            return;
+        }
+        
+        _permissionGroupSearchText = value;
+        _groups = _permissionGroupSearchText.IsNullOrWhiteSpace() ? _allGroups : _allGroups.Where(x => x.DisplayName.Contains(_permissionGroupSearchText, StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
It should search for inner items too. When I search for `Page` keyword, it should display CmsKit in the left side since it contains Page Management feature. @ebicoglu suggested like this

![permission-search](https://github.com/user-attachments/assets/0af923f5-8b97-4fdd-a0e5-92d7e38285e8)